src/ContainerCommitModal.jsx

```diff
@@ -67,9 +67,10 @@ class ContainerCommitModal extends React.Component {
         });
     }
 
-    // TODO
     handleCommit() {
-
+        const commitMsg = {...this.state};
```

Comment:
commitMsg is used only here. I think it makes sense to use this.state directly here.